OPEN_SOURCE_POLICY.md

```diff
@@ -0,0 +1,163 @@
+# Open Source Policy
```

Comment:
Shouldn't this file have the SPDX headers? :D

Reply:
Oh, it does. I'll add it and then track down why there was no error from the CI.

Reply:
@josevalim Added.

Reply:
No CI error: https://github.com/oss-review-toolkit/ort-ci-github-action/pull/45

Will open an update PR once this is merged.